golem-cli/src/model/mod.rs

```diff
@@ -122,6 +122,44 @@ impl From<String> for ProjectName {
     }
 }
 
+#[derive(Clone, PartialEq, Eq, Debug)]
+pub enum ProjectReference {
```

Comment:
This is used together with the existing ProjectName type. The meanings are now

ProjectName -> Just a simple project name
ProjectReference -> Our project slug syntax with either {account_email}/{project_name} or plain {project_name}